Dockerfile

```diff
@@ -38,23 +38,38 @@ FROM clearlinux:base
 LABEL maintainers="Intel"
 LABEL description="PMEM CSI Driver"
 
-# update and install needed bundles:
+# update and install needed bundles/packages:
 # file - driver uses file utility to determine filesystem type
 # xfsprogs - XFS filesystem utilities
-# storge-utils - for lvm2 and ext4(e2fsprogs) utilities
+# e2fsprogs- Ext filesystem utilities
+# LVM2-bin - for lvm2 and ext4(e2fsprogs) utilities
 ARG CACHEBUST
-RUN swupd update && swupd bundle-add file xfsprogs storage-utils && rm -rf /var/lib/swupd
+ENV PATH="/bin:/usr/bin"
+#NOTE: This below RUN command is intentionally combined all commands, to flatten the docker layers
+RUN swupd update && swupd bundle-add mixer && \
+# Create custom 'pmem-csi' bundle that holds all the needed packages
+    mixin package add file --bundle pmem-csi && \
+    mixin package add xfsprogs --bundle pmem-csi && \
+    mixin package add e2fsprogs-extras --bundle pmem-csi && \
+    mixin package add LVM2-bin --bundle pmem-csi && \
+    mixin build && \
```

Comment:
The mixin approach is an interesting idea. It's something that @mythi mentioned two weeks ago. Did you have the same idea independently? What a coincidence :smile: 

This is worth sharing with the Clear Linux stacks team. They run into similar size issues with the existing bundles.

Reply:
> The mixin approach is an interesting idea. It's something that @mythi mentioned two weeks ago. Did you have the same idea independently? What a coincidence 😄
> 
I was googling around "micro bundles" then I came across mixer and mixin. :-) But mixer is such a huge bundle, around 1.5 GB it increasing the build timing.